docs/src/commands/index.md

```diff
@@ -3,14 +3,15 @@
 `wasm-pack` has several commands to help you during the process of building
 a Rust-generated WebAssembly project.
 
-- `new`: This command generates a new project for you using a template. [Learn more][generate]
+- `new`: This command generates a new project for you using a template. [Learn more][new]
 - `build`: This command builds a `pkg` directory for you with compiled wasm and generated JS. [Learn more][build]
-- `pack` and `publish`: These command will create a tarball, and optionally publish it to a registry, such as npm. [Learn more][pack-pub]
-
+- `pack` and `publish`: These commands will create a tarball, and optionally publish it to a registry, such as npm. [Learn more][pack-pub]
 
 ### Deprecated Commands
+
 - `init`: This command has been deprecated in favor of `build`.
 
+[new]: ./new.html
```

Comment:
this looks like it is already listed! could you remove the redundant line?

Reply:
done